src/audio-player-composable/components/forward-button/forward-button.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {ButtonOverrides, ButtonSize} from '../../../button/types';
+import {IconButton} from '../../../icon-button';
+import {useTheme} from '../../../theme';
+import {filterOutFalsyProperties} from '../../../utils/filter-object';
+import {withOwnTheme} from '../../../utils/with-own-theme';
+import {useAudioPlayerContext} from '../../context';
+import {AudioPlayerIconButtonProps} from '../../types';
+import defaults from './defaults';
+
+const ThemelessAudioPlayerForwardButton: React.FC<AudioPlayerIconButtonProps> = React.memo(
+  ({overrides, ...props}) => {
+    const {getForwardButtonProps} = useAudioPlayerContext();
+
+    const theme = useTheme();
+
+    const audioPlayerForwardOverrides: ButtonOverrides = {
+      ...theme.componentDefaults.audioPlayerForwardButton,
+      ...filterOutFalsyProperties(overrides),
+    };
+
+    const propsFromContex =
+      getForwardButtonProps! && getForwardButtonProps(props);
+    return (
+      <IconButton
+        data-testid="audio-player-forward-button"
+        size={props.size || ButtonSize.Medium}
```

Comment:
```suggestion
        size={ButtonSize.Medium}
```
that will do the same job, 
If the user provide their own size prop it will be inside "props" and will override that one.